### PR TITLE
Add pipelines to repository

### PR DESCRIPTION
**Pull request by ausias-armesto**
## Release Notes

- **New Features**
	- Introduced automated GitHub workflows for Build, Merge PR, and Close release processes, enhancing version management and artifact accessibility.
	- Implemented a CI/CD pipeline for building and publishing the application across multiple Node.js versions.

- **Bug Fixes**
	- Improved file management by updating `.gitignore` to exclude auto-generated files from version control.

- **Documentation**
	- Added detailed documentation for GitHub workflows in the project repository.
	- Restructured the README to focus on contributing through pull requests, removing specific deployment instructions.

- **Chores**
	- Removed outdated workflow files to streamline the CI/CD process.

### CODE REVIEW

**Comment by esterlus** [comment]
Can we get the node version matrix? Or will that come later?

File: .github/workflows/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# GitHub Workflows
+
+This document describes the GitHub workflows used in this project.
```
Comment (esterlus) [inline .github/workflows/README.md:3]: Any reason those should not be part of the main README? Or at least linked there?

**Review by esterlus — CHANGES_REQUESTED** [review]
For me the biggest issue is, that deployment is documented good enough in the main README.md.

Can you update the relevant sections of the README accordingly?

File: .github/workflows/README.md (hunk 1 above)
Comment (ausias-armesto) [reply .github/workflows/README.md:3]: I considered it as internal development process documentation, which is different from the generic repository documentation which is more oriented to show how to use the repository.
But I don't have a strong argument on this, I will add the link

**Comment by ausias-armesto** [comment]
> Can we get the node version matrix? Or will that come later?

Implemented already

**Comment by ausias-armesto** [comment]
> For me the biggest issue is, that deployment is documented good enough in the main README.md. This should also include the new Changelog handling.
> 
> Can you update the relevant sections of the README accordingly?

Changelog is auto-generated automatically from the PR merged into main. No need to do anything from the development side other than contribute always to main from a PR. The changelog will be included in the body of the Github  Release

**Review by esterlus — APPROVED** [review]
Let's try the new workflow then

**Pull request merged**